src/Streamly/Internal/Data/Fold.hs

```diff
@@ -108,6 +108,9 @@ module Streamly.Internal.Data.Fold
     -- ** Mapping
     , transform
     , map
+    , toStream
+    , toStreamRev
```

Comment:
This should be moved next to `toList`/`toListRev` under the `Collectors` section above.